board/fancom/hmi/Kconfig

```diff
@@ -0,0 +1,22 @@
+if TARGET_FANCOM_HMI
+
+config SYS_BOARD
+        default "hmi"
+
+config SYS_VENDOR
+        default "fancom"
```

Comment:
Maybe Fancom iso fancom

Reply:
Nope, because than have to rename the directory in boards/.